snuba/datasets/configuration/entity_subscription_builder.py

```diff
@@ -1,31 +1,43 @@
 import logging
-from typing import Type
+from typing import Type, Union
 
 import sentry_sdk
 
 from snuba.datasets.configuration.json_schema import V1_ENTITY_SUBSCIPTION_SCHEMA
 from snuba.datasets.configuration.loader import load_configuration_data
 from snuba.datasets.entity_subscriptions.pluggable_entity_subscription import (
-    PluggableEntitySubscription,
+    PluggableEntitySubscriptionComplex,
+    PluggableEntitySubscriptionSimple,
 )
 
 logger = logging.getLogger("snuba.entity_subscriptions_builder")
 
 
 def build_entity_subscription_from_config(
     file_path: str,
-) -> Type[PluggableEntitySubscription]:
+) -> Type[Union[PluggableEntitySubscriptionComplex, PluggableEntitySubscriptionSimple]]:
     config = load_configuration_data(
         file_path, {"entity_subscription": V1_ENTITY_SUBSCIPTION_SCHEMA}
     )
+
     with sentry_sdk.start_span(
         op="build", description=f"Entity Subscription: {config['name']}"
     ):
+        # Default to simple entity subscription
+        PluggableEntitySubscription: Type[
+            Union[PluggableEntitySubscriptionComplex, PluggableEntitySubscriptionSimple]
+        ] = PluggableEntitySubscriptionSimple
+
+        if "type" in config and config["type"] == "complex":
```

Comment:
I don't think we need this check or the `type` field. If the configuration has one of the `aggregations` fields below, we know it's complex. Otherwise it's simple.